index.html

```diff
@@ -173,7 +173,47 @@ <h1 class="header">What is Technica?</h1>
             <p>At Technica, we want to challenge our hackers to step out of their comfort zone and try something new, whether that's exploring a new technology, tinkering with hardware, or coding for the very first time! At Technica, we want our hackers to learn, grow, and meet new people, so join us at Technica 2021!</p>
           </div>
           <div class="col-lg-4">
-              <img id="about-image" src="img/hackers-illustration.png" alt="hackers posing in four squares of a polaroid" width="100%" height="100%">
+            <img id="about-image" src="img/hackers-illustration.png" alt="hackers posing in four squares of a polaroid" width="100%" height="100%">
+          </div>
+        </div>
+      </div>
+    </div>
+
+    <div class="what-can-you-do section" id="what-can-you-do">
+      <div class="container">
+        <h1 class="header">What You Can Do at Technica</h1>
+        <div class="slide"></div>
+        <ul class="nav nav-tabs">
+          <li class="tab-container">
+            <a data-toggle="tab" class="tab active orange" href="#friends">
+              <div class="title orange"><div class="tab active show radio orange"></div>
+                <span>Make new friends</span>
```

Comment:
hmm slightly bigger headings might look better bc right now they look small in proportion to the box  - no changes needed now, but flagging for reevaluation when we add the content

Reply:
Noted <3